src/navigator-card-body.jsx

```diff
@@ -280,7 +287,24 @@ export const NavigatorCardBody = ({
                 <Spinner />
             </Flex>
         );
-    if (isGrid) {
+    const clearFilter = () => {
+        setCurrentFilter("");
+    };
+    if (sortedFiles.length === 0) {
+        return (
+            <EmptyState>
+                <EmptyStateHeader
+                  titleText="No results found" headingLevel="h4"
+                  icon={<EmptyStateIcon icon={SearchIcon} />}
+                />
+                <EmptyStateFooter>
+                    <EmptyStateActions>
+                        <Button variant="link" onClick={clearFilter}>Clear search</Button>
```

Comment:
The clearing of the search filter should happen in the SearchInput

https://www.patternfly.org/components/text-input-group#utilities-and-icon